op-challenger/game/fault/preimages/large.go

```diff
@@ -34,7 +35,49 @@ func (p *LargePreimageUploader) UploadPreimage(ctx context.Context, parent uint6
 	// todo(proofs#467): split up the preimage into chunks and submit the preimages
 	//                   and state commitments to the preimage oracle contract using
 	//                   `PreimageOracle.addLeavesLPP` (`_finalize` = false).
+	uuid, err := p.newUUID()
+	if err != nil {
+		return fmt.Errorf("failed to generate UUID: %w", err)
+	}
+	err = p.initLargePreimage(ctx, uuid, data.OracleOffset, uint32(len(data.OracleData)))
+	if err != nil {
+		return fmt.Errorf("failed to initialize large preimage with uuid: %s: %w", uuid, err)
+	}
 	// todo(proofs#467): track the challenge period starting once the full preimage is posted.
 	// todo(proofs#467): once the challenge period is over, call `squeezeLPP` on the preimage oracle contract.
-	return errNotSupported
+	return nil
```

Comment:
Seems like this should still return not supported since we didn't complete the upload?